packages/core/src/components/group/group.tsx

```diff
@@ -78,6 +79,8 @@ export class Group {
 
   @Element() hostElement!: HTMLIxGroupElement;
 
+  itemSelected = false;
```

Comment:
`itemSelected` is used inside `reactive typescript` part here is missing a `@State`